README.md

```diff
@@ -1,2 +1,92 @@
-# RFCs
-Holds RFCS (“Request for Comments”), proposals to improve Enarx
+# RFCs for Enarx
+
+This repo holds Request for Comments (RFCs), enhancement proposals to improve
+the Enarx project.
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the
+[normal GitHub pull request workflow](https://github.com/enarx/enarx/wiki/How-to-contribute).
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the community and
+the [sub-team]s.
+
+If you are here to learn about Enarx, as the code is still young, we
+recommend you look at the [code][https://github.com/enarx/enarx/) and
+[wiki](https://github.com/enarx/enarx/wiki) in the
+[main Enarx repo](https://github.com/enarx/enarx/).
+
+In the future, you will be able to browse [the RFC Index](index.md) for
+a current listing of all RFCs and their statuses.
+
+There are 2 types of Enarx RFCs:
+
+* RFCs that describe individual features (in the [features](./features) folder)
+* RFCs that explain concepts underpinning many features
+(in the [concepts](./concepts) folder)
+
+## RFC Lifecycle
+
+RFCs go through a standard lifecycle.
+
+![lifecycle](lifecycle.png)
+
+#### PROPOSED
+To __propose__ an RFC, [use these instructions to raise a PR](
+contributing.md#how-to-propose-an-RFC) (Pull Request) against the repo.
+Proposed RFCs are considered a "work in progress", even after they are merged.
+In other words, they haven't been endorsed by the community yet, but they
+seem like reasonable ideas worth exploring.
+
+#### DEMONSTRATED
+__Demonstrated__ RFCs have one or more implementations available, listed in
+the "Implementations" section of the RFC document. As with the PROPOSED status,
+demonstrated RFCs haven't been endorsed by the community, but the ideas put
+forth have been more thoroughly explored through the implementation(s).
+The demonstrated status is an __optional__ step in the lifecycle.
+
+#### ACCEPTED
+To get an RFC __accepted__, [build consensus](contributing.md#how-to-get-an-RFC-accepted)
+for your RFC on [chat](https://gitter.im/enarx/community) and in community
+meetings. If your RFC is a feature that's code-related, it MUST have
+reasonable tests. An accepted RFC is incubating on a standards track;
+the community has decided to polish it and is exploring or pursuing implementation.
+
+#### ADOPTED
+To get an RFC __adopted__, [socialize and implement](contributing.md#how-to-get-an-rfc-adopted).
```

Comment:
I think we should put the word "it" after "implement"

Reply:
Makes sense, adding this, thanks.